src/Mutator/Util/MutatorProfile.php

```diff
@@ -59,6 +59,7 @@
         Mutator\Arithmetic\PowEqual::class,
         Mutator\Arithmetic\ShiftLeft::class,
         Mutator\Arithmetic\ShiftRight::class,
+        Mutator\Rounding\ChangeRoundFunctionCall::class,
```

Comment:
This needs to be updated to the current class name.

Since this is a new group(profile), please add it to a new const. See other groups as an example